### PR TITLE
Statistics react library

### PR DESCRIPTION
**Pull request by NektariosFifes**
**Description**

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: ReactGoStat/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "dependencies": {
+    "@geist-ui/react": "latest",
+    "@nektariosfifes/practical-react-ui-kit": "npm:@NektariosFifes/practical-react-ui-kit@^1.0.1",
```
Comment (jonaslagoni) [inline ReactGoStat/package.json:31]: Generally, it is highly discouraged to use custom external libraries unless there is a good reason to do so. Why are you using a private library?

File: Async-Simulator/package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Async-Simulator",
+  "version": "1.0.0",
```
Comment (jonaslagoni) [inline Async-Simulator/package.json:3]: Lets start with `"version": "0.1.0",` 😄

File: ReactGoStat/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        indent: 0,
+        '@typescript-eslint/indent': 0,
+        '@typescript-eslint/member-delimiter-style': 0,
+    },
+    settings:  {
+        react:  {
+            version:  'detect',
+        },
+    },
+};
```
Comment (jonaslagoni) [inline ReactGoStat/.eslintrc.js:26]: I would suggest we use somewhat the same `eslint` configuration as other react repositories have such as  https://github.com/asyncapi/generator-react-sdk/

File: ReactGoStat/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+## ReactGoStat
+### In Early Development
```
Comment (jonaslagoni) [inline ReactGoStat/README.md:2]: I would like to have some more general information about the setup of the library. It doesn't have to be super specific, but it should be enough for me to get a clear overview of the structure of the project.

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Awesome 👍 
Just gonna add a general review now and then deep dive later. Here are the general comments:
- I am missing some tests which ensure correctness now and in the future.
- Remove all console logs if possible, unless there is a good reason to keep them.
- Specific comments.

File: ReactGoStat/package.json (hunk 1 above)
Comment (NektariosFifes) [reply ReactGoStat/package.json:31]: yes this was a packaged version i was testing out i will remove it

Comment (NektariosFifes) [reply ReactGoStat/package.json]: sry i by accident hit request review thinking it was refresh.
 I dont understand i pushed to my fork the correction yet i dont know if the pull has been modified

**Pull request closed without merging**